hub/powertoys/color-picker.md

```diff
@@ -65,75 +65,75 @@ Color Picker has following settings:
 - Activation shortcut
 - Behavior of activation shortcut
 - The default format of a copied color (HEX, RGB, etc.)
-- Show the name of the color. The shown color name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
+- Show the name of the color. The shown name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
 - Order and presence of color formats in the editor
 
 ![ColorPicker Settings screenshot.](../images/pt-colorpicker-settings.gif)
 
+To add a new color format, select **Add custom color format**.
+
 To reorder the colors, select the **...** button on the right and select **Move up** or **Move down**.
 
-To disable a color format, toggle the switch on the right. Disabled color formats are not deleted but also not shown in the Color Picker Editor window.
+To disable a color format, toggle the switch on the right. Disabled color formats are not deleted but are not shown in the Color Picker Editor window.
```

Comment:
```suggestion
To disable a color format, turn off the toggle next to that color format. Disabled color formats are not deleted but are not shown in the Color Picker editor window.
```

Change the way that [toggles are described to be in line with the style guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui).

Removed the capital letter from 'Editor' to be consistent with the editor section earlier on the page.